File: src/natives/csharp/source/IPC/SystemHostRequest.cs
using Repeat.ipc;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Repeat.IPC {
    public class SystemHostRequest : RequestGenerator {

        public SystemHostRequest(RepeatClient client) : base(client) {
            this.Type = "system_host";
            this.Device = "system";
        }

        public bool KeepAlive() {
            Action = "keep_alive";
            ClearParams();

            return SendRequest(blockingWait: false) == null ? false : true;
        }
    }
}

File: src/natives/python/system_host_request.py
import request_generator


class SystemHostRequest(request_generator.RequestGenerator):
    def __init__(self, client):
        super(SystemHostRequest, self).__init__(client)
        self.type = 'system_host'
        self.device = 'system'
        
    def keep_alive(self):
        self.action = 'keep_alive'
        self.params = []
        return self.send_request(blocking_wait = False)